docs/api/process.md

```diff
@@ -7,3 +7,7 @@ upstream node:
 * `process.versions['electron']` String - Version of Electron.
 * `process.versions['chrome']` String - Version of Chromium.
 * `process.resourcesPath` String - Path to JavaScript source code.
+
+## process.hang
+
+Call in the `renderer` process to trigger `unresponsive` event.
```

Comment:
But there wont any browserwindow creation at all for the event to be triggered, which i believed to be the use of this method ?